src/cmd/wallet.rs

```diff
@@ -121,7 +126,14 @@ fn list_keychain_addresses(
                     .get(script)
                     .map(|utxos| Amount::from_sat(utxos.iter().map(|utxo| utxo.txout.value).sum()))
                     .unwrap_or(Amount::ZERO);
+                let txn_count = match txn_map.get(script) {
+                    Some(txn_vec) => txn_vec.len() as u64,
+                    None => 0,
+                };
 
+                if unused && (txn_count != 0) {
```

Comment:
unnecessary parens. The whole reason I looked into clippy was to catch this but it looks like that has just broken the whole build.